rust_src/src/process.rs

```diff
@@ -39,6 +52,7 @@ fn get_process(name: LispObject) -> LispObject {
         name
     } else {
         name.as_string_or_error();
+        ();
```

Comment:
What's this for?

Reply:
It was a place holder. I meant to remove it.